frontend/app/utils/api.tsx

```diff
@@ -21,6 +21,9 @@ type GetExtractorQueryKey = [string, string, boolean]; // [queryKey, uuid, isSha
 
 type OnSuccessFn = (data: { uuid: string }) => void;
 
+axios.defaults.withCredentials = true;
```

Comment:
@bracesproul I will need your help to figure out how to do this correctly later on. For now going to merge to get unblocked

Reply:
The other way to do it would be via the config object when actually performing the axios request:

```typescript
const response = await axios.get(`${baseUrl}/shared/extractors/${uuid}`, {
  withCredentials: true,
});
```